examples/expectations/regex_based_column_map_expectation_template.py

```diff
@@ -0,0 +1,78 @@
+"""
+This is a template for creating custom ColumnMapExpectations.
```

Comment:
Docstring needs to be updated from ColumnMapExpectation to RegexBasedColumnMapExpectations...